pkgs/by-name/ex/exam-monitor/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchurl, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  pname = "exam-monitor";
+  version = "24044";
+
+  src = fetchurl {
+    url = "https://login.exammonitor.dk/exam.jar";
```

Comment:
This url does not enforce the version, meaning the hash will break in the future. Please replace it with a archive.org url